app/models/category.rb

```diff
@@ -0,0 +1,4 @@
+class Category < ApplicationRecord
+  validates :name, presence: true
+  validates :name, length: { minimum: 4, maximum: 30 }
```

Comment:
`minimum: 4` sounds very strict. I can imagine a category with 3 characters. Let's say somebody collects spendings related to the IT environment and wants to add a category called `IDE`.

Reply:
Okay, that's sounds fair enough. I will change that in a minute.